atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -142,6 +153,22 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji connection established")
 
+    def _fetch_log_files(self):
+        try:
+            namespace = get_build_json().get('metadata', {}).get('namespace', None)
```

Comment:
The usage of `.get` won't cause `KeyError` exception. However, `get_build_json` could raise `KeyError`. It's probably cleaner to rely on `KeyError` for both cases and just do: `namespace = get_build_json()['metadata']['namespace']`

Reply:
Fixed.